projects/vanilla/src/app/configurators/facet-configurator.component.ts

```diff
@@ -25,8 +26,16 @@ import { ConfiguratorContext, ComponentConfig } from "@sinequa/ngx-ui-builder";
   <ng-container *ngIf="config.type !== 'facet-date'">
     <uib-checkbox [context]="context" property="parameters.searchable" label="Searchable"></uib-checkbox>
     <uib-checkbox [context]="context" property="parameters.allowOr" label="Allow Multi-selecting with OR"></uib-checkbox>
-    <uib-checkbox [context]="context" property="parameters.allowAnd" label="Allow Multi-selecting with AND" *ngIf="config.type !== 'facet-tree'"></uib-checkbox>
+    <uib-checkbox [context]="context" property="parameters.allowAnd" label="Allow Multi-selecting with AND" *ngIf="!isTree"></uib-checkbox>
```

Comment:
En fait rien n'empêche de faire un AND avec la facette tree (même si c'est inhabituel car normalement un document ne peut pas se trouver à deux treepaths différents)